### PR TITLE
[DOCS] fixed duplication of text in code example

### PR DESCRIPTION
**Pull request by kyleaton**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
-fixed duplication of text in code example
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md [hunk 1]
```diff
@@ -104,31 +104,6 @@ from flytekit import Resources, kwtypes, task, workflow
 from flytekit.types.file import CSVFile
 from flytekitplugins.great_expectations import GreatExpectationsTask
```
Comment (NathanFarmer) [inline docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md]: These imports are also duplicated

File: docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md [hunk 2]
```diff
@@ -104,31 +104,6 @@ from flytekit import Resources, kwtypes, task, workflow
...
-) -> int:
-   file_task_object(dataset=dataset)
-   return len(pd.read_csv(dataset))
-
-
-@workflow
-def file_wf(
-   dataset: CSVFile = "https://raw.githubusercontent.com/superconductive/ge_tutorials/main/data/yellow_tripdata_sample_2019-01.csv",
-) -> int:
-   return file_task(dataset=dataset)
```
Comment (NathanFarmer) [inline docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md]: This `return` statement is missing from line 103

**Review by NathanFarmer — CHANGES_REQUESTED** [review]
A few minor changes needed!

File: docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md (hunk 2 above)
Comment (kyleaton) [reply docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md]: https://github.com/great-expectations/great_expectations/pull/3503/commits/ef5cab6565c4910a727ee79c7283510f785ceeba should cover this

File: docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md (hunk 1 above)
Comment (kyleaton) [reply docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md]: https://github.com/great-expectations/great_expectations/pull/3503/commits/ef5cab6565c4910a727ee79c7283510f785ceeba Should cover this

**Review by NathanFarmer — APPROVED** [review]
LGTM!

**Pull request merged**